client/reader/following-manage/index.jsx

```diff
@@ -96,10 +92,19 @@ class FollowingManage extends Component {
 			debounce( this.resizeSearchBox, 50 )
 		);
 		this.resizeSearchBox();
+
+		// this is a total hack. In React-Virtualized you need to tell a WindowScroller when the things
+		// above it has moved with a call to updatePosision().  Our issue is we don't have a good moment
+		// where we know that the content above the WindowScroller has settled down and so instead the solution
+		// here is to call updatePosition in a regular interval. the call takes about 0.1ms from empirical testing.
+		this.updatePosition = setInterval( () => {
+			this.windowScrollerRef && this.windowScrollerRef.updatePosition();
+		}, 100 );
```

Comment:
just thinking 10 times a second feels like a lot. maybe every 300ms and instead of doing the actual work, we schedule it with requestAnimationFrame? or maybe i'm just being paranoid.

how long does the work take on a phone?